DESCRIPTION

```diff
@@ -0,0 +1,32 @@
+Package: popgenInfo
+Title: popgenInfo is not really an R package, but this is here to list the dependencies for running all the vignettes.
+Version: 0.1
+Authors@R: person("First", "Last", email = "first.last@example.com",
```

Comment:
Did you leave this intentionally "blank" (so as to be filled in later by everyone, for example?), or was this an oversight?